### ISSUE TITLE
Failing in command prompt

### ISSUE THREAD

**Issue by Advik-B** [shawwn/supports-color-python#1]
```py
Traceback (most recent call last):
  File "O:\python-code\Compiler\testclr.py", line 1, in <module>
    from supports_color import supportsColor
  File "O:\python-code\Compiler\venv\lib\site-packages\supports_color\__init__.py", line 203, in <module>
    stdout=createSupportsColor(sys.stdout),
  File "O:\python-code\Compiler\venv\lib\site-packages\supports_color\__init__.py", line 191, in createSupportsColor
    level = _supportsColor(stream, streamIsTTY=stream and stream.isatty(), **options)
  File "O:\python-code\Compiler\venv\lib\site-packages\supports_color\__init__.py", line 166, in _supportsColor
    if re.search(r'-256(color)?$', env.get('TERM'), re.IGNORECASE):
  File "C:\Programs\Python310\lib\re.py", line 200, in search
    return _compile(pattern, flags).search(string)
TypeError: expected string or bytes-like object
```

**Comment by shawwn** [comment]
Okay, this should be fixed as of version `0.1.3`, which I've just deployed. Does anyone want to run `pip install -U supports-color` and verify? cc: @SilenZcience @Advik-B

**Comment by SilenZcience** [comment]
at this point i can verify that it works on both ubuntu and windows (command prompt & powershell) without crashes, but i do not have a windows version prior to truecolor support ...
